lib/TeleIrc.js

```diff
@@ -180,6 +180,26 @@ class TeleIrc {
   initStage3_initBots(ircbot, tgbot) {
     this.ircbot = ircbot;
     this.tgbot = tgbot;
+
+    this.ircbot.connect(() => {
+      this.nickservIdentify();
+    });
+  }
+
+  /**
+   * Identifies to NickServ, if password is provided.
+   */
+  nickservIdentify() {
+    if ((this.config.irc.nickservPassword.length > 0) &&
+        (this.config.irc.nickservService.length > 0)) {
+      console.log("Identifying to NickServ...");
+      this.ircbot.say(
+        this.config.irc.nickservService,
+        "IDENTIFY " + this.config.irc.nickservPassword
```

Comment:
In some cases, the bot has issues "claiming" a nick (where `my-bot` becomes `my-bot1`, `my-bot2`, etc.), so the given nickname may be different than a registered nick. We have not figured out why this happens yet.

Could we add a fixed configuration parameter for a NickServ username, different from the bot name? The syntax to send to a NickServ account would be:

`IDENTIFY <NickServ username> <NickServ password>`